src/user/introduction.rst

```diff
@@ -9,14 +9,18 @@ A brief introduction
 What is conda-forge?
 --------------------
 
-Conda-forge is a community effort that provides conda packages for a wide range of software.
+``conda-forge`` is a community effort that provides conda packages for a wide range of software.
 
-**Missing a package that you would love to install with conda?** - Chances are we have already packaged it for you!
+**Missing a package that you would love to install with conda?** - Chances are we have already packaged it for you! You can `search <https://anaconda.org/>`__ for packages online as well as keep an eye out for packages from our ``conda-forge`` organization.
 
-You can `search <https://anaconda.org/>`__ for packages online. Look out for packages provided by our ``conda-forge`` organization.
-
-**Cannot find a package or only outdated versions of a package?** - Everybody is welcome to contribute to our package stack! Please refer to :ref:`becoming_involved`, for an overview on how to start contributing.
+We value all kinds of contributions - not just code. A few recommended ways to start contributing to conda-forge are:
```

Comment:
```suggestion
We value all kinds of contributions — not just code. A few recommended ways to start contributing to ``conda-forge`` are:
```

The dash here should be an em dash (—) not a single dash. Or you can go for three separate dashes (---)